hoth_api.yaml

```diff
@@ -319,5 +319,351 @@ components:
       oneOf:
         - $ref: "#/components/schemas/DescriptionOfWork"
     DescriptionOfWork:
-      description: "TODO: AT-7341, define DescriptionOfWork schema"
+      description: "Schema used to generate DescriptionOfWork document"
       type: object
+      properties:
+        to_title:
+          type: string
+          maxLength: 60
+        scope:
+          type: string
+          maxLength: 300
+        scope_surge:
+          type: integer
+          maximum: 50
+        current_environment: # DOW section 3
+          $ref: "#/components/schemas/CurrentEnvironment"
+        performance_requirements: # DOW section 4 (and its subtasks)
+          $ref: "#/components/schemas/PerformanceRequirements"
+        # DOW section 5 TBD
+        # DOW section 6 TBD
+        period_of_performance: # DOW section 7
+          $ref: "#/components/schemas/PeriodOfPerformance"
+        # DOW section 8 TBD
+        gfe_overview: # DOW section 9
+          $ref: "#/components/schemas/GFEOverview"
+        contract_considerations: # DOW section 10 (and its subtasks)
+          $ref: "#/components/schemas/ContractConsiderations"
+        section_508_accessibility_standards_for_cloud_computing: # DOW section 11
+          $ref: "#/components/schemas/SensitiveInformation"
+    CurrentEnvironment:
+      description: "Part of DOW, encapsulates all data for 3. Background/Current Environment section."
+      properties:
+        current_environment_exists:
+          type: boolean
+        environment_instances:
+          type: array
+          items:
+            $ref: "#/components/schemas/EnvironmentInstance"
+          maxItems: 4000
+        additional_info:
+          type: string
+          maxLength: 1000
+    EnvironmentInstance:
+      description: "Part of DOW, represents a single Environment Instance"
+      properties:
+        instance_name:
+          type: string
+          maxLength: 100
+        classification_level:
+          $ref: "#/components/schemas/ClassificationLevel"
```

Comment:
This doesn't necessarily need to match other APIs' naming; however, we've been sorta all over the place with `ClassificationLevel` or `SecurityDomain` or other terminology. @jeffsegal-ccpo thoughts on how/whether this can/should match the CSP API?